lib/android.js

```diff
@@ -726,15 +726,14 @@ function findSDK(dir, config, androidPackageJson, callback) {
 		return callback(true);
 	}
 
-	const proguardPath = path.join(dir, 'tools', 'proguard', 'lib', 'proguard.jar'),
-		emulatorPath = path.join(dir, 'emulator', 'emulator' + exe),
+	const emulatorPath = path.join(dir, 'emulator', 'emulator' + exe),
```

Comment:
non-blocking: Consider template literal instead of string appending.

```suggestion
	const emulatorPath = path.join(dir, 'emulator', `emulator${exe}`),
```